crates/anvil/src/hardfork.rs

```diff
@@ -135,7 +135,7 @@ impl From<EthereumHardfork> for SpecId {
             EthereumHardfork::Cancun | EthereumHardfork::Latest => Self::CANCUN,
             EthereumHardfork::Prague => Self::PRAGUE,
             // TODO: switch to latest after activation
-            EthereumHardfork::PragueEOF => Self::PRAGUE_EOF,
+            EthereumHardfork::PragueEOF => Self::PRAGUE,
```

Comment:
Could you expand on the origin of this change? I'm missing some context

It looks like EOF is not going to part of the first Prague hardfork

Reply:
Fixing PragueEOF was renamed to OSAKA